pyogrio/tests/test_geopandas_io.py

```diff
@@ -186,6 +186,79 @@ def test_read_fids_force_2d(test_fgdb_vsi):
         assert not df.iloc[0].geometry.has_z
 
 
+#@pytest.mark.filterwarnings("ignore: Layer")
```

Comment:
```suggestion
@pytest.mark.filterwarnings("ignore: Layer")
```

(needed for `LIMIT` / `skip_features` test below)